scripts/migration/migrate-unix.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+set -e
+
+echo "********************************************"
+echo "      Ferdium User Data Migration Tool      "
+echo "          WARNING: UNIX OS Only!            "
+echo "********************************************"
+echo "\n\nThis tool migrates your user data from Ferdi to Ferdium."
+echo "Intended to be used on Linux or MacOS machine.\n\n"
+read -p "Do you want to proceed? (y/N) " confirm
+
+case "$confirm" in
+  y|Y|[yY][eE][sS] ) echo "Starting...";;
+  n|N|[nN][oO] ) exit 1;;
+  * ) exit 1;;
+esac
+
+os_check=$(uname)
+
+if [ $os_check == "Linux" ]; then
+  echo "Your OS is: Linux"
+  BASE_PATH="$HOME/.config"
+elif [ $os_check == "Darwin" ]; then
+  echo "Your OS is: MacOS"
+  BASE_PATH="$HOME/Library/Application Support"
+else
+  echo "Your OS is not supported by this script"
+  exit 1
+fi
+
+FERDI_PATH="$BASE_PATH/Ferdi"
+FERDIUM_PATH="$BASE_PATH/Ferdium2"
+
+if [ -d "$FERDIUM_PATH" ]; then
+  echo "\nPath $FERDIUM_PATH exist, making a backup\n"
+  if ! mv -vf "$FERDIUM_PATH" "$FERDIUM_PATH.bak" 2> /dev/null ; then
+    read -p "A previous backup already exists at $FERDIUM_PATH.bak. do you want to remove it? (y/N) " confirm
+    echo
+    case "$confirm" in
+      y|Y|[yY][eE][sS] ) echo "Deleting...";;
+      n|N|[nN][oO] ) exit 1;;
+      * ) exit 1;;
+    esac
+    echo
+    rm -rf "$FERDIUM_PATH.bak"
+    mv -vf "$FERDIUM_PATH" "$FERDIUM_PATH.bak"
+  fi
+fi
+
+if mv -vf "$FERDI_PATH" "$FERDIUM_PATH"; then
```

Comment:
@Evan-aja - this line fails for me on macos. @SpecialAro - can you please test this if you have a mac? If not,  I will spend time on this later this evening

Reply:
my bad - testing was incorrect. Now all good